src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -95,6 +94,11 @@ trait Backend {
    */
   def stdoutStderr(descriptor: WorkflowDescriptor, callName: String, index: ExecutionIndex): CallLogs
 
+  /**
+   * Provides a function that given a WdlFile, returns its hash.
```

Comment:
I see my confusion, I was picturing this as Workflowdeacriptor=>file harsher without realizing the latter was the function referred to be the provides statement